libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1063,6 +1063,64 @@ void TypeChecker::endVisit(EmitStatement const& _emit)
 	m_insideEmitStatement = false;
 }
 
+/**
+ * Creates a tuple declaration syntax from a vector of variable declarations.
+ *
+ * @param decls a tuple of variables
+ *
+ * @returns a Solidity language confirming string of a tuple variable declaration.
+ */
+static string createTupleDecl(vector<VariableDeclaration const*> const & decls)
+{
+	if (decls.size() == 1)
+	{
+		VariableDeclaration const* var = decls[0];
+		string const typeName = var->annotation().type->toString(true);
+		return typeName + " " + var->name();
+	}
+
+	stringstream sb;
+
+	sb << '(';
+
+	for (size_t i = 0, e = decls.size(); i != e; ++i)
```

Comment:
I think it is safer and more idiomatic to use `for (size_t i = 0; i < decls.size(); ++i)` - don't assume the compiler wil not optimize that.